Add a basic README

chrismytton opened the pull request:
Fill out a basic README.

Fixes #1 
# Notes to reviewer

I struggled a bit to come up with a decent description of this gem. I'm not sure if that's because it's not solving a well enough defined problem or if it's just a bit late on a Friday but any suggestions for improvements are welcome!

chrismytton committed:
Add a basic README

chrismytton commented:
@tmtmtmtm 👀

tmtmtmtm commented on README.md:
```diff
@@ -1,8 +1,7 @@
-
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method.
```
My main comment here is that this is only really explaining the _what_, rather than the _why_.

chrismytton replied:
Yep, good point! I've added a few more words on why we built it and how we're using it.

chrismytton committed:
Expand README to include why people might use it

Rather than just saying what it is give a bit more background on what
we're using it for and why other people might be interested in using
it.

tmtmtmtm commented on README.md:
```diff
@@ -1,8 +1,10 @@
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method. We ([EveryPolitician](http://everypolitician.org/))
+are using this library in our scrapers to make them more declaritive. It allows
```
typo: declaritive → declarative

chrismytton replied:
Oops! Fixed in https://github.com/everypolitician/field_serializer/pull/11/commits/be28beb9eff91aabea5872422f837cb62f0e6c95.

tmtmtmtm reviewed (changes requested):
Tiny typo, and I suspect we're going to want to tweak this considerably over time, but it's definitely an improvement over the existing README, so otherwise 👍

chrismytton committed:
Fix type declaritive → declarative

tmtmtmtm committed:
Merge branch 'master' into fill-out-readme
